discovery-infra/test_infra/tools/assets.py

```diff
@@ -44,6 +50,110 @@ def release_all(self) -> None:
         self._release()
 
 
-class NetworkAssets(Assets):
+class LibvirtNetworkAssets(Assets):
+    """ An assets class that stores values based on the current available
+        resources, in order to allow multiple installations while avoiding
+        conflicts. """
+
     def __init__(self):
         super().__init__(assets_file=consts.TF_NETWORK_POOL_PATH)
+        self._allocated_ips_objects = []
+        self._allocated_bridges = []
+
+    def get(self) -> Munch:
+        asset = deepcopy(super().get())
```

Comment:
from my point of view you don't need this, you need json object with starting points. Not sure what it gives to you to use asset class here? there is no point to release object even as it is released by removing vms.
I think using here assets.get is making this more complicated